### PR TITLE
Private Game Seeds Data

### PR DESCRIPTION
**Pull request by ttakahashi1591**
## Description

Please include a description of what was changed
- Add seed data for a Private Game

## Type of change

- [ ] fix
- [ ] feat
- [ ] test
- [ ] refactor
- [ ] docs

## Checklist

- [ ] code has been self reviewed
- [ ] code runs without any errors
- [ ] thorough testing has been implemented if adding feature
- [ ] all tests pass

### Thanks!

### CODE REVIEW

File: lib/tasks/daily_vote_processing.rake [hunk 1]
```diff
@@ -283,5 +283,26 @@ namespace :populate_database do
 
     @round.process_round
     @round2.process_round
+
+    @private_game1 = Game.create!(length_in_days: 151, guess_lead_time: 3, player_cap: 315, game_type: 0, results: nil)
```
Comment (mbkuhl) [inline lib/tasks/daily_vote_processing.rake:287]: This needs to be game type 1

File: lib/tasks/daily_vote_processing.rake [hunk 2]
```diff
@@ -283,5 +283,26 @@ namespace :populate_database do
...
+
+    @user_game1 = UserGame.create!(user_id: @user1.id, game_id: @private_game1.id, invitation: accepted)
+    @user_game2 = UserGame.create!(user_id: @user2.id, game_id: @private_game1.id, invitation: accepted)
```
Comment (mbkuhl) [inline lib/tasks/daily_vote_processing.rake:290]: accepted needs to be :accepted

File: lib/tasks/daily_vote_processing.rake [hunk 3]
```diff
@@ -283,5 +283,26 @@ namespace :populate_database do
...
+      lon: data11[:location][:lon],
+      maxtemp_f: data11[:weather_data][:maxtemp_f],
+      mintemp_f: data11[:weather_data][:mintemp_f],
+      maxwind_mph: data11[:weather_data][:maxwind_mph],
+      totalprecip_in: data11[:weather_data][:totalprecip_in],
+      avgvis_miles: data11[:weather_data][:avgvis_miles],
+      avghumidity: data11[:weather_data][:avghumidity],
+      daily_chance_of_rain: data11[:weather_data][:daily_chance_of_rain],
+      daily_chance_of_snow: data11[:weather_data][:daily_chance_of_snow])
+    @round.update(close_date: (Date.today - 4).to_s, process_date: (Date.today - 1).to_s)
```
Comment (mbkuhl) [inline lib/tasks/daily_vote_processing.rake:306]: I think this is good.

**Review by mbkuhl — CHANGES_REQUESTED** [review]
I went ahead and tested this in rails c to double check. Game type 0 is competitive game. For the user games, without the colon to symbolize, it will be looking for a variable names accepted.

**Review by bkchilidawg — APPROVED** [review]

File: lib/tasks/daily_vote_processing.rake (hunk 1 above)
Comment (ttakahashi1591) [reply lib/tasks/daily_vote_processing.rake:287]: Thank you for than clarification. I have this updated and will push up once I get the other updates cleared.

File: lib/tasks/daily_vote_processing.rake (hunk 2 above)
Comment (ttakahashi1591) [reply lib/tasks/daily_vote_processing.rake:290]: Done!

**Review by mbkuhl — APPROVED** [review]
Perfect

**Pull request merged**